core/tests/karma.conf.js

```diff
@@ -102,7 +102,7 @@ module.exports = function(config) {
     customLaunchers: {
       Chrome_Travis: {
         base: 'Chrome',
-        flags: ['--no-sandbox']
+        flags: ['--window-size=650,650']
```

Comment:
Ah is this from some previous PR?

Reply:
No, I needed this to get the chrome size less than 760px so that it could be checked for mobile sites

Reply:
Hmm, I have a feeling that this is breaking some things. I've restarted your tests multiple times and they all fail on the same test. Will it be possible to keep the `--no-sandbox` tag?

Reply:
@WickedBrat I think you can't do this either, it affects all tests.

See if, instead, you can mock the relevant service in your Karma test so that it affects only your Karma test. There are plenty of resources online, and some examples in the codebase, of how to do this.